Profiles/README.md

```diff
@@ -1,6 +1,6 @@
 # Profiles
 Fill in the following details about yourself below:
 
-- Your name
-- A picture of yourself
-- A link to your favourite [Pudding article](https://pudding.cool/) or music video
\ No newline at end of file
+- Darya Dyachkova
+- pic of Dasha
+- https://www.youtube.com/watch?v=lmTmGLzPVyM
```

Comment:
To add a link in markdown to a text you can use the following [title of link] (link)  (ignore the space)